OmkeshwarGupta/task4.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=, initial-scale=1.0">
+    <title>Nevbar</title>
+    <link rel="stylesheet" href="https://cc-bhu.github.io/web-development/assets/css/navbar.css">
+    <style>
+        section {
+
+            margin-top: 5%;
+            margin-bottom: 50%;
+            text-align: center;
+            padding-top: 5%;
+            font-family: 'Times New Roman', Times, serif;
+
+
+
+        }
```

Comment:
I suggest you to try height: 100vh next time, which means 100% vertical height.
It will allow you to set height according to screen sizes.

Reply:
Thankyou! Next time I will try that.